games-sports/xmoto/xmoto-0.6.1-r101.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..3} )
+
+inherit cmake lua-single
+
+DESCRIPTION="A challenging 2D motocross platform game, where physics play an important role"
+HOMEPAGE="https://xmoto.tuxfamily.org"
+SRC_URI="https://github.com/xmoto/xmoto/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="double-precision +nls"
+
+REQUIRED_USE="${LUA_REQUIRED_USE}"
+
+RDEPEND="app-arch/bzip2
+	dev-db/sqlite:3
+	dev-games/ode[double-precision=]
+	${LUA_DEPS}
+	dev-libs/libxdg-basedir
+	dev-libs/libxml2:=
+	media-fonts/dejavu
+	media-libs/libjpeg-turbo
```

Comment:
Probably should add `:=` here.

Reply:
You mean for some other dependency other than libxml2? 

If there are other dependency fixes then it would be good opportunity to do those at the same time. Feel free to cherrypick.

Reply:
I just meant for media-libs/libjpeg-turbo because you added it in f01035c8db0f387f7c256fcf3f154dca0c38df2d.

Reply:
Alright, forgot I did that. That was before I decided against fixing all the preexisting CI failures. Addressed